assembl/static2/js/app/reducers/adminReducer/landingPage.js

```diff
@@ -82,16 +88,103 @@ export const modulesByIdentifier: ModulesByIdentifierReducer = (state = initialS
   }
 };
 
+const initialPage = Map({
+  _hasChanged: false,
+  titleEntries: List(),
+  subtitleEntries: List(),
+  buttonLabelEntries: List(),
+  headerImage: Map({
+    externalUrl: '',
+    mimeType: '',
+    title: ''
+  }),
+  logoImage: Map({
+    externalUrl: '',
+    mimeType: '',
+    title: ''
+  })
+});
+type PageState = Map<string, any>;
+type LandingPageReducer = (PageState, ReduxAction<Action>) => PageState;
+export const page: LandingPageReducer = (state = initialPage, action) => {
+  switch (action.type) {
+  case UPDATE_LANDING_PAGE: {
+    let newState = state;
+    if (action.page.headerImage) {
+      newState = newState
+        .setIn(['headerImage', 'title'], action.page.headerImage.title)
```

Comment:
`newState.set('headerImage', fromJS(action.page.headerImage)` is shorter (same for `logoImage`)